keps/sig-network/2030-topology-aware-proxying/README.md

```diff
@@ -245,6 +253,138 @@ not reliant on any other feature being enabled in any other release. [KEP
 #2004](https://github.com/kubernetes/enhancements/issues/2004) will be dependent
 on this KEP though.
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [x] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: TopologyAwareProxying
+    - Components depending on the feature gate: kube-proxy (also used by KEP
+      2004 for kube-controller-manager)
+
+* **Does enabling the feature change any default behavior?**
+  Not really. Kube-Proxy will still consume endpoints without these topology
+  labels (the default). When these labels are set (a separate opt-in discussed
```

Comment:
Let's remove 2004-related references here and below.

We should purely focus on what is proposed in this KEP for the purpose of this PRR.

Reply:
Good call, removed